img/o/opensea.io.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><path fill="#2081e2" d="M50.4.5a49.6 49.6 0 10-.6 99.2A49.6 49.6 0 0050.4.5z"/><path d="M35.6 25.9a23.2 23.2 0 011.2 26.7H20.2l15.4-26.7z"/><path d="M86.3 58.5c0 .2-.1.4-.3.5-1.1.5-4.8 2.2-6.4 4.4-4 5.5-7 14.3-13.8 14.3H37.3c-10.1 0-18.5-8-18.4-18.6 0-.3.2-.5.5-.5h13.4c.5 0 .8.4.8.8V62c0 1.4 1.1 2.5 2.5 2.5h10.2v-5.9h-7a29.8 29.8 0 00-1.5-38.7c3 .3 5.8.9 8.4 1.7v-1.7c0-1.7 1.4-3.1 3.1-3.1 1.7 0 3.1 1.4 3.1 3.1v4c9.5 4.4 15.8 11.8 15.8 20.2a19 19 0 01-5.8 13.3c-.7.7-1.7 1.1-2.7 1.1h-7.2v5.9h9c1.9 0 5.4-3.7 7.1-5.9l.3-.2 16.6-3.8c.3-.1.7.2.7.5l.1 3.5z"/></svg>
```

Comment:
The icon had something wrong with the dimensions of the blue circle, and the ship should be white.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><circle cx="50" cy="50" r="50" fill="#2081e2"/><path fill="#fff" d="M86.3 58.5c0 .2-.1.4-.3.5-1.1.5-4.8 2.2-6.4 4.4-4 5.5-7 14.3-13.8 14.3H37.3c-10.1 0-18.5-8-18.4-18.6 0-.3.2-.5.5-.5h13.4c.5 0 .8.4.8.8V62c0 1.4 1.1 2.5 2.5 2.5h10.2v-5.9h-7a29.8 29.8 0 00-1.5-38.7c3 .3 5.8.9 8.4 1.7v-1.7c0-1.7 1.4-3.1 3.1-3.1 1.7 0 3.1 1.4 3.1 3.1v4c9.5 4.4 15.8 11.8 15.8 20.2a19 19 0 01-5.8 13.3c-.7.7-1.7 1.1-2.7 1.1h-7.2v5.9h9c1.9 0 5.4-3.7 7.1-5.9l.3-.2 16.6-3.8c.3-.1.7.2.7.5zM35.6 25.9a23.2 23.2 0 0 1 1.2 26.7H20.2"/></svg>
```

Reply:
Wow thank you! I didn't even notice that, it shows up correctly on my computer. 🤣